apis/sts/src/main/java/org/jclouds/aws/filters/FormSignerV4.java

```diff
@@ -101,7 +105,25 @@
       this.serviceAndRegion = serviceAndRegion;
    }
 
-   @Override public HttpRequest filter(HttpRequest request) throws HttpException {
+   private String getAnnotatedApiVersion(HttpRequest request) {
```

Comment:
[minor] Although private could you make return `Optional<String>` instead of null ?